internal/pkg/levee/testdata/src/levee_analysistest/example/tests/callorder/colocation.go

```diff
@@ -15,56 +15,31 @@
 package callorder
 
 import (
+	"fmt"
+	"io"
```

Comment:
I actually wrote this in IntelliJ, and I'm not seeing this import re-grouping behavior. I do have a "Run goimports and save" macro bound to `Ctrl + s`, so that may be why I'm seeing the goimports behavior.

I think goimports is standard, so we should stick to that as much as possible. If I see spurious import diffs in PRs you make, I won't mind. I don't think it's reasonable to expect all contributors to open and save their files in IntelliJ before sending out a PR. :thinking:

Reply:
> I actually wrote this in IntelliJ, and I'm not seeing this import re-grouping behavior. I do have a "Run goimports and save" macro bound to `Ctrl + s`, so that may be why I'm seeing the goimports behavior.

Yeah, I figured it might be a personal configuration issue.  Or maybe a Community / Ultimate mismatch?  But probably just I have a box checked somewhere or an inspection turned on that you don't.  Not a big deal.

> I think goimports is standard, so we should stick to that as much as possible. If I see spurious import diffs in PRs you make, I won't mind. If we want to enforce IntelliJ formatting, I think we would need contributors to open and save each file in IntelliJ before sending out a PR, which I don't think is reasonable. 🤔

Strictly speaking, support for at least "add local imports last" is built into `goimports`, we would just need to be explicit that we want `goimports -local github.com/google/go-flow-levee,levee_analysistest`.  Easily something we could cook into our CI. 
 But it's probably not worth it in the end, so I'm happy to leave it at no harm no foul.

Reply:
I might.  Either way, out of scope for this PR.  Good to talk about it though.

Marking this thread resolved.